File: backend/Ronaldo.GestaoDeClientes/Ronaldo.GestaoDeClientes.Core/Aggregates/ClientAggregate/Entities/Address.cs
using Ronaldo.GestaoDeClientes.Core.SharedKernel.Entities;
using System.Collections.Generic;

namespace Ronaldo.GestaoDeClientes.Core.Aggregates.ClientAggregate.Entities
{
    public class Address : Entity
    {
        public string PostalCode { get; set; }
        public string PublicPlace { get; set; }
        public int Number { get; set; }
        public string Complement { get; set; }
        public string Neighborhood { get; set; }
        public string Locality { get; set; }
        public string State { get; set; }
        public virtual IEnumerable<Client> Clients { get; set; }
    }
}

File: backend/nodejs_mongodb/gestao-de-clientes/app/models/address/address.js
var mongoose = require('mongoose');
var Schema = mongoose.Schema;

var AddressSchema = new Schema({
    id: "",
    postalCode: "",
    publicPlace: "",
    number: "",
    complement: "",
    neighborhood: "",
    locality: "",
    state: "",
    active: ""
});

module.exports = mongoose.model('Address', AddressSchema);